drivers/soundwire/intel_init.c

```diff
@@ -190,58 +232,19 @@ static struct sdw_intel_ctx
 		if (!(link_mask & BIT(i)))
 			continue;
 
-		/* Alloc and init devices */
-		ldev = kzalloc(sizeof(*ldev), GFP_KERNEL);
-		if (!ldev)
-			goto err;
-
 		/*
-		 * keep a handle on the allocated memory, to be used in all other functions.
-		 * Since the same pattern is used to skip links that are not enabled, there is
-		 * not need to check if ctx->ldev[i] is NULL later on.
+		 * init and add devices
```

Comment:
maybe a "." at the end of "init and add devices" I was confused if the second line was a continuation of the first.

Reply:
reworded.